docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -0,0 +1,126 @@
+# `autonomy mint`
+
+Tools to mint components on chain
+
+### Options
+
+`--use-ethereum`
+: Use `ethereum` to chain profile to interact with the contracts.
+
+`--use-goerli`
+: Use `goerli` to chain profile to interact with the contracts.
+
+`--use-custom-chain`
+: Use custom-chain to chain profile to interact with the contracts.
+
+`--use-local`
+: Use local to chain profile to interact with the contracts.
+
+`--skip-hash-check`
+: Skip hash check when verifying dependencies on chain
+
+### Chain selection
```

Comment:
Why is this info in a separate section and not in the introduction/description of the command?